packages/cloudflare-optimizer-scripts/test/index.test.js

```diff
@@ -105,11 +129,62 @@ describe('handleEvent', () => {
     expect(output).toBe(`transformed-${input}`);
   });
 
+  it('should handle redirects opaquely', async () => {
+    const mockedResponse = new Response('', {
+      status: 302,
+      statusText: '',
+    });
+
+    const event = {
+      request: {method: 'GET', url: 'https://example.com/'},
+      respondWith: jest.fn(),
+      passThroughOnException: jest.fn(),
+    };
+
+    global.fetch.mockImplementation(() => Promise.resolve(mockedResponse));
+
+    handleEvent(event, defaultConfig);
+    expect(await event.respondWith.mock.calls[0][0]).toBe(mockedResponse);
+    expect(event.respondWith).toHaveBeenCalledTimes(1);
+  });
+
+  it('should rewrite location header for redirects to origin in proxy mode', async () => {
```

Comment:
I think I'm confused -- how is this test different from the one on Line 74 with the title `should rewrite Location header for Redirect in response to Non-Get Requests (Proxy Mode)`?

Reply:
There are two usage sites, since non GET methods have different forwarding logic I am making sure that the Redirect headers are rerwriten in that case as well. 

Is that in our domain of responsibilities?